### PR TITLE
Add Streamis 0.3.0

### PR DESCRIPTION
**Pull request by Alexkun**
### What is the purpose of the change
Add Streamis 0.3.0, including changes from 0.2.0 to 0.3.0

Related issues/PRs
Related issues: 
#68 #69 #70 #71 #72 #73 #74 #75 #76 #77 #78 #79 #80 #81 #82 #83 #84 #85 #86 #87 #88 #89 #90

### Brief change log
(for example:)
- Add support for checkpoint of flink.
- Add support for detach manage mode, etc.

### Verifying this change
(Please pick either of the following options)  
This change is a trivial rework / code cleanup without any test coverage.  
(or)  
This change is already covered by existing tests, such as (please describe tests).  
(or)  
This change added tests and can be verified as follows:  
(example:)  
- Added tests for creating and execute the Streamis jobs and verify the availability of different Streamis Job, such as flinkSQL, Jar.

### Does this pull request potentially affect one of the following parts:
- Dependencies (does it add or upgrade a dependency): (yes / no)
- Anything that affects deployment: (yes / no / don't know)
- The Core framework, i.e., JobManager, Server.: (yes / no)

### Documentation
- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: README-ZH.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Dimension节点、Transform节点、Sink节点 和 [Visualis](https://github.com
 ![流式应用配置](docs/images/stream_job_config_1.png)
 ![流式应用配置](docs/images/stream_job_config_2.png)
 
-&nbsp; &nbsp; &nbsp; &nbsp;更多功能，请参考：[Streamis 用户手册](docs/zh_CN/0.2.0/使用文档/Streamis用户手册.md)。
+&nbsp; &nbsp; &nbsp; &nbsp;更多功能，请参考：[Streamis 用户手册](docs/zh_CN/0.2.5-webank/使用文档/Streamis用户手册.md)。
```
Comment (wushengyeyouya) [inline README-ZH.md:49]: Should change the branch name.

File: README-ZH.md [hunk 2]
```diff
@@ -73,14 +73,14 @@ Dimension节点、Transform节点、Sink节点 和 [Visualis](https://github.com
 
 ## 编译和安装部署
 
-&nbsp; &nbsp; &nbsp; &nbsp;请参考 [Streamis 安装部署文档](docs/zh_CN/0.2.0/Streamis安装文档.md) ，用于安装部署 Streamis 及其依赖环境。
+&nbsp; &nbsp; &nbsp; &nbsp;请参考 [Streamis 安装部署文档](docs/zh_CN/0.2.5-webank/Streamis安装文档.md) ，用于安装部署 Streamis 及其依赖环境。
```
Comment (wushengyeyouya) [inline README-ZH.md:76]: Should change the branch name.

File: README-ZH.md [hunk 3]
```diff
@@ -73,14 +73,14 @@ Dimension节点、Transform节点、Sink节点 和 [Visualis](https://github.com
...
 
 
 ----
 
 ## 示例和使用指引
 
-&nbsp; &nbsp; &nbsp; &nbsp;请到 [用户使用文档](docs/zh_CN/0.2.0/使用文档/Streamis用户手册.md) ，了解如何快速使用 Streamis。
+&nbsp; &nbsp; &nbsp; &nbsp;请到 [用户使用文档](docs/zh_CN/0.2.5-webank/使用文档/Streamis用户手册.md) ，了解如何快速使用 Streamis。
```
Comment (wushengyeyouya) [inline README-ZH.md:83]: Should change the branch name.

File: README.md [hunk 4]
```diff
@@ -71,7 +71,7 @@ The Dimension node, Transform node, Sink node and [Visualis](https://github.com/
 
 ## Compile and install deployment
 
-please refer to [Streamis Installation and Deployment Document](docs/en_US/0.2.0/StreamisDeployment.md) for installing and deploying Streamis.
+please refer to [Streamis Installation and Deployment Document](docs/en_US/0.2.5-webank/StreamisDeployment.md) for installing and deploying Streamis.
```
Comment (wushengyeyouya) [inline README.md:74]: Should change the branch name.

File: bin/upgrade.sh [hunk 5]
```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-# Use to upgrade from 0.1.0 to 0.2.0
+# Use to upgrade from 0.1.0 to 0.2.5-webank
```
Comment (wushengyeyouya) [inline bin/upgrade.sh:18]: Should change the branch name.

**Review by wushengyeyouya — COMMENTED** [review]
Cantains lots of 0.2.5, 0.2.5-webank and 0.3.1, 0.3.1-webank, please update it.

File: README-ZH.md (hunk 3 above)
Comment (Alexkun) [reply README-ZH.md]: changed file location

File: README.md (hunk 4 above)
Comment (Alexkun) [reply README.md]: Updated.

File: bin/upgrade.sh (hunk 5 above)
Comment (Alexkun) [reply bin/upgrade.sh:18]: This script should not change.

File: README.md [hunk 6]
```diff
@@ -71,7 +71,7 @@ The Dimension node, Transform node, Sink node and [Visualis](https://github.com/
 
 ## Compile and install deployment
 
-please refer to [Streamis Installation and Deployment Document](docs/en_US/0.2.0/StreamisDeployment.md) for installing and deploying Streamis.
+please refer to [Streamis Installation and Deployment Document](docs/en_US/0.3.0-webank/StreamisDeployment.md) for installing and deploying Streamis.
```
Comment (wushengyeyouya) [inline README.md:74]: Please modify it.

File: bin/upgrade025.sh [hunk 7]
```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-# Use to upgrade from 0.1.0 to 0.2.0
+# Use to upgrade from 0.1.0 to 0.2.5-webank
```
Comment (wushengyeyouya) [inline bin/upgrade025.sh:18]: Please modify it.

File: bin/upgrade025.sh [hunk 8]
```diff
@@ -39,7 +39,7 @@ interact_echo(){
   done
 }
 
-interact_echo "Are you sure the current version of Streamis is 0.1.0 and need to upgrade to 0.2.0 ?"
+interact_echo "Are you sure the current version of Streamis is 0.2.x < 0.2.5 and need to upgrade to 0.2.5 ?"
```
Comment (wushengyeyouya) [inline bin/upgrade025.sh:42]: Please modify it.

File: docs/en_US/0.2.5/StreamisDeployment.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
 # Streamis installation and deployment documentation
 
 ## 1. Component introduction
-Streamis0.2.0 provides the Streamis-JobManager component, the role of the component is <br>
+Streamis0.2.5-webank provides the Streamis-JobManager component, the role of the component is <br>
```
Comment (wushengyeyouya) [inline docs/en_US/0.2.5/StreamisDeployment.md:4]: Please modify it.

**Review by wushengyeyouya — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (Alexkun) [reply README.md]: done

File: bin/upgrade025.sh (hunk 7 above)
Comment (Alexkun) [reply bin/upgrade025.sh]: done

File: bin/upgrade025.sh (hunk 8 above)
Comment (Alexkun) [reply bin/upgrade025.sh]: done

File: docs/en_US/0.2.5/StreamisDeployment.md (hunk 9 above)
Comment (Alexkun) [reply docs/en_US/0.2.5/StreamisDeployment.md]: done

**Review by Alexkun — COMMENTED** [review]
updated

File: docs/en_US/0.2.5/StreamisDeployment.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # Streamis installation and deployment documentation
 
 ## 1. Component introduction
-Streamis0.2.0 provides the Streamis-JobManager component, the role of the component is <br>
+Streamis0.2.5 provides the Streamis-JobManager component, the role of the component is <br>
```
Comment (wushengyeyouya) [inline docs/en_US/0.2.5/StreamisDeployment.md:4]: Please modify it.

File: docs/en_US/0.2.5/StreamisDeployment.md [hunk 11]
```diff
@@ -156,10 +156,10 @@ sudo nginx -s reload
 
 ## 5. Access to DSS
 
-If you want to use the streamis0.2.0 front end normally, you also need to install the DSS StreamisAppConn plug-in. Please refer to: [StreamisAppConn plug-in installation document](development/StreamisAppConnInstallationDocument.md)
+If you want to use the streamis0.2.5 front end normally, you also need to install the DSS StreamisAppConn plug-in. Please refer to: [StreamisAppConn plug-in installation document](development/StreamisAppConnInstallationDocument.md)
 
 ## 6. Linkis Flink engine compilation and installation
-If you want to run streamis0.2.0 normally, you also need to install the linkis Flink engine. Please refer to: [linkis Flink engine installation document](https://linkis.apache.org/zh-CN/docs/1.1.2/engine_usage/flink/)
+If you want to run streamis0.2.5 normally, you also need to install the linkis Flink engine. Please refer to: [linkis Flink engine installation document](https://linkis.apache.org/zh-CN/docs/1.1.2/engine_usage/flink/)
```
Comment (wushengyeyouya) [inline docs/en_US/0.2.5/StreamisDeployment.md:162]: Please modify it.

**Review by wushengyeyouya — CHANGES_REQUESTED** [review]

File: docs/en_US/0.2.5/StreamisDeployment.md (hunk 10 above)
Comment (Alexkun) [reply docs/en_US/0.2.5/StreamisDeployment.md]: done

File: docs/en_US/0.2.5/StreamisDeployment.md (hunk 11 above)
Comment (Alexkun) [reply docs/en_US/0.2.5/StreamisDeployment.md]: done

**Review by wushengyeyouya — APPROVED** [review]
LGTM.

**Pull request merged**